src/libraries/Directory.Build.targets

```diff
@@ -219,6 +219,14 @@
   <Target Name="GetDocumentationFile"
           Returns="$(DocumentationFile)"/>
 
+  <Choose>
```

Comment:
I don't speak MSBuild too well, so there might be a better way to do this. Adding @safern here, in case he sees some unintended consequences of adding these lines.

Reply:
The `Choose`/`When` is like a `switch` statement. Since this is just using one condition, there is no need to do that. You could just put the condition in the property declaration itself. The reason why we did it this way for the INTERNAL_NULLABLE_ATTRIBUTES below, is because we're setting an ItemGroup as well so this is the best way to share this condition. If you wanted, you could use the `Choose` below and add another `When` statement. However this is more complicated that it should be for this case, I would just add the `DefineConstants` declaration here with the condition inline:

https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.targets#L16

```xml
<DefineConstants Condition="'$(IsPrerelease)' != 'false' AND '$(TargetFramework)' != '$(NetCoreAppCurrent)'">$(DefineConstants),USE_INTERNAL_ACCESSIBILITY</DefineConstants>
```